tests/testthat/test-name_repair.R

```diff
@@ -92,7 +92,7 @@ test_that("check_unique() errors for empty or duplicated names", {
 
 test_that("unique_names() is idempotent", {
   x <- c("..20", "a..1", "b", "", "a..2")
-  expect_identical(unique_names(x), unique_names(unique_names(x)))
+  expect_identical(unique_names(!!x), unique_names(unique_names(!!x)))
```

Comment:
Can you elaborate on your motivation for this?

Reply:
By doing this, the output for the failing test included `unique_names(c("..20", "a..1", "b", "", "a..2"))` . When this test fails the next time, it will be easier to troubleshoot.